samples/README.md

```diff
@@ -0,0 +1,39 @@
+# Open Job Description Sample Templates
+
+This directory contains a set of sample templates for the Open Job Description standard.
+The aim with these is to provide a small sampling of just some of the things that you
+can do with Open Job Description, and a reference set of demonstrations of template
+concepts that you can incorporate in to your own templates.
+
+All samples are provided AS-IS. We strongly recommend that you 
```

Comment:
We strongly recommend that you test all samples thoroughly before use, and customize the samples as required by your individual needs.